pkg/localmetrics/localmetrics.go

```diff
@@ -0,0 +1,146 @@
+package localmetrics
+
+import (
+	"fmt"
+
+	managedv1alpha1 "github.com/openshift/rbac-permissions-operator/pkg/apis/managed/v1alpha1"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	clusterScope   = "cluster"
+	namespaceScope = "namespace"
+)
+
+var (
+	log = logf.Log.WithName("localmetrics_grouppermission")
+
+	// RBACClusterwidePermissions for cluster-wide permissions
+	RBACClusterwidePermissions = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "rbac_permissions_operator_cluster_permission",
+		Help: "Configured permissions in the cluster-wide scope",
+	}, []string{
+		"group_name",
+		"group_permission_name",
+		"cluster_permission_name",
+		"state",
+	})
+
+	// RBACNamespacePermissions for per-namespace permissions
+	RBACNamespacePermissions = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "rbac_permissions_operator_namespace_permission",
+		Help: "Configured permissions in a per-namespace scope",
+	}, []string{
+		"group_name",
```

Comment:
This is a lot of labels. If anyone these change value that's a new permission. Are all of these needed?

Reply:
@fahlmant Right now the labels are mapping to the attributes in the CR for the namespace permission. It's possible we find over time they're not all needed, but for now, I think they're needed. That said, I could see removing "state" as it could be seen as redundant.